### DISCUSSION TITLE
Some commands can't be used in command log

### DISCUSSION THREAD

**Discussion by cool-RR** [saulpw/visidata#1985] [Q&A] [answered]
I want to have a workflow in which I open a sheet with VisiData, and it does a bunch of commands before I'm presented with the sheet for usage. I figured I'd use the `--play` command line argument with a `.vdj` file that has all of my commands.
  
  I noticed that some commands don't really work in this setup, like `search-col`. I tried this file:
  
  ```
  #!vd -p
  {"longname": "open-file", "input": "foo.csv", "keystrokes": "o"}
  {"longname": "search-col", "input": "2"}
  ```
  
  When I launch `vd --play 1.vdj`, it does open `foo.csv` but it does not do the `search-col` command. Is this a known limitation? What can I do?

**Comment by saulpw** [comment, marked as answer]
Hm, that should work!  Try adding `"col": "<colname>"` to the search-col line maybe?

**Reply by cool-RR** [reply]
Ah, this failed because of `setup_post_load_hook` in my `.visidatarc`. I changed it a bit to give up after 0.5 seconds. What would be really nice is if https://github.com/saulpw/visidata/issues/1681 would be implemented and I won't need it anymore, because it's an ugly hack anyway.

**Reply by cool-RR** [reply]
Could you try whether `zoom-cursor` works for you in a vdj file? Because it doesn't work for me, even when I disable my `.visidatarc` file.
